packages/tooltip/Tooltip.tsx

```diff
@@ -9,34 +9,47 @@ const TooltipTypes = {
   bottom: 'bottom',
 };
 
-type TooltipTypesType = typeof TooltipTypes;
+type TooltipTypesType = keyof typeof TooltipTypes;
```

Comment:
we dont need that suffix `Typ` in name. We now that it is a type alias already ;)